packages/webview_flutter/webview_flutter_wkwebview/darwin/webview_flutter_wkwebview/Sources/webview_flutter_wkwebview/StructWrappers.swift

```diff
@@ -0,0 +1,17 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Foundation
+
+/// Wrapper around `URLRequest`.
+///
+/// Since `URLRequest` is a struct, it is pass by value instead of pass by reference. This makes
+/// it not possible to modify the properties of a struct with the typical ProxyAPI system.
+class URLRequestWrapper {
```

Comment:
Interesting FYI for anyone curious and additional context that I can add to the comment if needed:

The Swift `URLRequest` represents the Objective-C `NSURLRequest` and `MutableNSURLRequest` and converts to one or the other depending on the context. 

This is different from `URL` and `NSURL`, which doesn't have a mutable equivalent. So a `URL` would always convert to the same `NSURL` instance when needed.

This distinction might come up when wrapping other structs.